File: oim-jruby/add_users_to_group.rb
require 'java'
require 'xlclient'

include_class('java.lang.Exception') {|package,name| "J#{name}" }
include_class 'java.lang.System' 
include_class 'java.util.HashMap'
include_class('Thor.API.tcUtilityFactory') {|package,name| "OIM#{name}"}


xlclient = XLAPIClient.new
xlclient.defaultLogin
grpIntf = xlclient.getUtility('grp')

    
grpKeys = []
usrLogin = 'BBB16'
#grpKey = xlclient.getGrpKey(grpName)
usrKey = xlclient.getUsrKey(usrLogin)

puts "usrKey = #{usrKey}"
grpKeys.each do |grpKey|
    puts "grpKey = #{grpKey}"
    grpIntf.addMemberUsers(grpKey, usrKey)
end

xlclient.close
System.exit 0

File: oim-jython/addUsersToGroup.py
from xlclient import *
from java.util import *
from java.lang import *

import sys

if __name__ == '__main__':
    xlclient = XLAPIClient('xelsysadm', 'xelsysadm')
    grpIntf = xlclient.grpIntf
    
    grpName = 'res2'
    grpKey = xlclient.getGrpKey(grpName)


    for i in range(4, 10011):
	print 'Adding user = ', str(i), ' to group ', grpName
	grpIntf.addMemberUser(grpKey, i)

    xlclient.close()
    sys.exit(0)
